src/TuDa.CIMS.Web/Components/Pages/ShoppingCartPage.razor.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.AspNetCore.Components;
+using MudBlazor;
+using TuDa.CIMS.Shared.Entities;
+using TuDa.CIMS.Web.Components.ShoppingCart;
+
+namespace TuDa.CIMS.Web.Components.Pages;
+
+public partial class ShoppingCartPage
+{
+    [Inject]
+    public required IDialogService DialogService { get; set; }
```

Comment:
🔧 To private and set it to `null!`
```suggestion
    private IDialogService _dialogService { get; set; } = null!;
```